### DISCUSSION TITLE
problema interfaccia Energia

### DISCUSSION THREAD

**Discussion by Ale1975-Iron** [indomus/forum#1698] [HUB: Home Assistant]
Ciao a tutti,

utilizzo uno shelly EM per monitorare i consumi e vorrei inserirlo nell'interfaccia Energia.
Con questa configurazione, lo trovo solo nelle statistiche consumo GAS. 
Cosa sbaglio?

Ho usato la stessa configurazione per gli Shelly PlugS e 1PM e riesco ad inserirli correttamente nella parte dispositivi individuali.
Spero di essere stato chiaro nel spiegare la mia necessità

```
  - platform: mqtt
    name: "Ass Home"
    state_topic: "shellies/shellyem-XXXXXX/emeter/0/power"
    value_template: "{{ value }}"
    unit_of_measurement: "W"
  - platform: integration
    source: sensor.ass_home
    name: "Consumo Home"
    unit_prefix: k
    round: 2    
```

**Comment by RobyBS** [comment]
io ne uso un paio .. per abitazione e per garage ... 
uso direttamente la loro integrazione ( shelly ) non dichiaro nulla in configuration.yaml ... 

poi vai su Impostazioni - Plance - Energia .. e sotto Rete Elettrica clicco su "aggiungi consumo" e selezioni l'entità di tuo interesse ... 
dopo qualche minuto ( forse anche 2 ore massimo ) ti apparirà il consumo sulla plancia relativa ... 


se selezioni il costo , avrai anche la voce di costo ...

spero di esserti stato utile .. 

se ho detto castronerie o esistono metodi più eleganti ... fatevi avanti !... :)

**Reply by Ale1975-Iron** [reply]
Grazie Roby per la risposta.
Tutti i miei Shelly plugs li ho inseriti proprio come hai detto tu… senza alcun problema.
È solo con lo shelly EM che non funziona.
Io ho integrato tutto con Mqtt ecco il motivo del codice.

Spero ci sia qualcun altro che abbia lo stesso problema ed abbia trovato la soluzione 😉

ciao e grazie ancora
